ovos_utils/system.py

```diff
@@ -48,15 +30,17 @@ def is_running_from_module(module_name):
             return True
     return False
 
-
 # system utils
+@deprecated("DEPRECATED: use ovos-PHAL-plugin-system", "0.2.0")
```

Comment:
If we're following semver starting here, these deprecation notices should target 1.0.0